### ISSUE TITLE
 scale background image such that it isn't obscured by the panel 

### ISSUE THREAD

**Issue by Zurd** [mate-desktop/mate-panel#352]
This issue is clearly described here:
https://bugzilla.gnome.org/show_bug.cgi?id=81037

In short, the mate panel is on top of a user's wallpaper. So we're all missing part of the wallpaper.

**Comment by Zurd** [comment]
Linux Mint 21.1 (Vera) with MATE Panel 1.26.2 and yes, the panel still cover the wallpaper.

**Comment by raveit65** [comment]
ohh, seems linuxmint is unusable  ....
I suggest to file out an issue at linuxmint to ask for developer working-hours.

**Comment by Zurd** [comment]
I don't think it is a Linux Mint bug, it's clearly in the code of MATE that somethings needs to be changed.

**Comment by raveit65** [comment]
Yes, and everybody can do a pull request, including members of linuxmint.
I am fedora maintainer and i don't have any free resources for issues from low interest.
....unsuscribe.
